### PR TITLE
Fix Nightly Tests for Binaries

### PR DESCRIPTION
**Pull request by mseth10**
## Description ##
This PR fixes nightly tests for binaries, that have been failing since 09/19.

One of the errors is because requirements file was not copied to the teat environment.
The other errors are due to numpy warnings treated as errors.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at https://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: tests/utils/notebook_test/__init__.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def run_notebook(notebook, notebook_dir, kernel=None, no_cache=False, temp_dir='
             nbformat.write(notebook, output_file)
             output_nb = io.open(output_file, mode='r', encoding='utf-8')
             for line in output_nb:
-                if "Warning:" in line:
+                if "Warning:" in line and "numpy operator signatures" not in line:
```
Comment (marcoabreu) [inline tests/utils/notebook_test/__init__.py]: @reminisce

**Comment by mseth10** [comment]
@ThomasDelteil @larroy ping for review

File: tests/utils/notebook_test/__init__.py (hunk 1 above)
Comment (reminisce) [reply tests/utils/notebook_test/__init__.py]: The warning message of `numpy operator signature` is for python version < 3.5. It does not affect functionality, but merely used for building numpy op documentation. I'm fine with this change.

**Comment by mseth10** [comment]
@marcoabreu can you please review and merge. Thanks.

File: ci/docker/Dockerfile.build.ubuntu_nightly_gpu [hunk 2]
```diff
@@ -29,6 +29,7 @@ COPY install/deb_ubuntu_ccache.sh /work/
 RUN /work/deb_ubuntu_ccache.sh
 
 COPY install/ubuntu_python.sh /work/
+COPY install/requirements /work/
```
Comment (ChaiBapchya) [inline ci/docker/Dockerfile.build.ubuntu_nightly_gpu]: so the test environment is in the '/work/` directory?

Comment (mseth10) [reply ci/docker/Dockerfile.build.ubuntu_nightly_gpu]: yes, work directory is created inside the docker and all required files are copied there.

**Pull request merged**